packages/button/src/LionButton.js

```diff
@@ -4,6 +4,8 @@ import { css, html, SlotMixin, DisabledWithTabIndexMixin, LitElement } from '@li
 const isKeyboardClickEvent = e => e.keyCode === 32 /* space */ || e.keyCode === 13; /* enter */
 // eslint-disable-next-line class-methods-use-this
 const isSpaceKeyboardClickEvent = e => e.keyCode === 32; /* space */
+// eslint-disable-next-line class-methods-use-this
```

Comment:
is that needed?
it's outside of the class right?

maybe we can even remove it from the other outside functions as well 🙈

Reply:
Removed it